evaluation/poets/NOTES

```diff
@@ -0,0 +1,24 @@
+RUN:
+pash run.sh (Runs all scripts in the scripts/ folder)
+
+Scripts that fail:
+    -> awk issues on pash?
+        scripts/bigrams_appear_twice.sh
+        scripts/find_anagrams.sh
+        scripts/vowel_sequencies_gr_1K.sh
+
+    -> trigram script does not exist
+        scripts/trigram_rec.sh
+
+Several scripts had different outputs in comparison with bash:
+    eg:
+        scripts/4letter_words.sh 
+    pash output: 
+        4767
```

Comment:
The issue seems to be grep -c. The way to address this would be to modify grep's annotation to consider `grep -c` to be pure instead of `stateless`.

Reply:
Adding @gyalpodongo here, who might be able to help with this.